### PR TITLE
[IMP] sale_manual_delivery: Avoid onchange function collisions

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Review by adrienpeiffer — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by ivantodorovich — APPROVED** [review]

**Review by cesar-tecnativa — APPROVED** [review]

**Review by CasVissers-360ERP — APPROVED** [review]

**Comment by CasVissers-360ERP** [comment]
I think this can be merged after 5 approvals?

**Comment by adrienpeiffer** [comment]
ping @OCA/crm-sales-marketing-maintainers

**Pull request merged**
